typings/index.d.ts

```diff
@@ -126,32 +132,38 @@ interface QueryByAPI {
 
 interface FindByAPI {
   findByText: (
-    text: string | RegExp,
-    waitForOptions?: WaitForOptions
+    text: TextMatch,
+    waitForOptions?: WaitForOptions,
+    options?: TextMatchOptions,
   ) => FindReturn;
   findByPlaceholderText: (
-    placeholder: string | RegExp,
-    waitForOptions?: WaitForOptions
+    placeholder: TextMatch,
+    waitForOptions?: WaitForOptions,
+    options?: TextMatchOptions,
   ) => FindReturn;
   findByDisplayValue: (
-    value: string | RegExp,
-    waitForOptions?: WaitForOptions
+    value: TextMatch,
+    waitForOptions?: WaitForOptions,
+    options?: TextMatchOptions,
   ) => FindReturn;
-  findByTestId: (testID: string | RegExp, waitForOptions?: WaitForOptions) => FindReturn;
+  findByTestId: (testID: TextMatch, waitForOptions?: WaitForOptions) => FindReturn;
   findAllByText: (
-    text: string | RegExp,
-    waitForOptions?: WaitForOptions
+    text: TextMatch,
+    waitForOptions?: WaitForOptions,
+    options?: TextMatchOptions,
   ) => FindAllReturn;
   findAllByPlaceholderText: (
-    placeholder: string | RegExp,
-    waitForOptions?: WaitForOptions
+    placeholder: TextMatch,
+    waitForOptions?: WaitForOptions,
+    options?: TextMatchOptions,
   ) => FindAllReturn;
   findAllByDisplayValue: (
-    value: string | RegExp,
-    waitForOptions?: WaitForOptions
+    value: TextMatch,
+    waitForOptions?: WaitForOptions,
+    options?: TextMatchOptions,
   ) => FindAllReturn;
   findAllByTestId: (
-    testID: string | RegExp,
+    testID: TextMatch,
     waitForOptions?: WaitForOptions
   ) => FindAllReturn;
```

Comment:
Looks like typescript types for findBy have incorrect options args order.

Reply:
Oops forgot to update, thanks!